File: exemplo_selenium_mult_linguagem/java/google_search.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class GoogleSearch {
	public static void main(String[] args) {
		WebDriver driver = new FirefoxDriver();
		driver.get("http://www.google.com");
		WebElement element = driver.findElement(By.name("q"));
		element.sendKeys("Hello Selenium WebDriver!");
		element.submit();
		System.out.println(driver.getTitle());
	}
}

File: exemplo_selenium_mult_linguagem/C#/google_search.cs
using System;
using OpenQA.Selenium;
using OpenQA.Selenium.Firefox;
using OpenQA.Selenium.Support.UI;

class GoogleSearch
{
static void Main()
	{
		IWebDriver driver = new FirefoxDriver();
		driver.Navigate().GoToUrl("http://www.google.com");
		IWebElement query = driver.FindElement(By.Name("q"));
		query.SendKeys("Hello Selenium WebDriver!");
		query.Submit();
		Console.WriteLine(driver.Title);
	}
}